安装ant-design-vue@2.0.0-rc.8的时候报错

- [x] I have searched the [issues](https://github.com/vueComponent/ant-design-vue/issues)         of this repository and believe that this is not a duplicate.
        
        
### Version
2.0.0-rc.8

### Environment
windows10,new Chrome，vue 3.0.4

### Reproduction link
[http://localhost](http://localhost)

### Steps to reproduce
rm -rf node_modules
npm cache clean --force
rm -rf package-lock.json
npm install

### What is expected?
不报错

### What is actually happening?
> ant-design-vue@2.0.0-rc.8 postinstall E:\work\xxxx\node_modules\ant-design-vue
> node scripts/postinstall || echo "ignore"

internal/modules/cjs/loader.js:968
throw err;
^

Error: Cannot find module 'node-emoji'
Require stack:
- E:\work\xxxx\node_modules\ant-design-vue\scripts\postinstall.js
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
at Function.Module._load (internal/modules/cjs/loader.js:841:27)
at Module.require (internal/modules/cjs/loader.js:1025:19)
at require (internal/modules/cjs/helpers.js:72:18)
at Object.<anonymous> (E:\work\xxxx\node_modules\ant-design-vue\scripts\postinstall.js:2:15)
at Module._compile (internal/modules/cjs/loader.js:1137:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
at Module.load (internal/modules/cjs/loader.js:985:32)
at Function.Module._load (internal/modules/cjs/loader.js:878:14)
at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12) {
code: 'MODULE_NOT_FOUND',
requireStack: [
'E:\\work\\xxxx\\node_modules\\ant-design-vue\\scripts\\postinstall.js'
]
}
"ignore"

---
怎么会有这种稀奇古怪的错误啊。虽然好像不影响使用，但是我有强迫症没办法

        

<!-- generated by issue-helper. DO NOT REMOVE -->

--- a/package.json
+++ b/package.json
@@ -152,7 +152,6 @@
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
     "mockdate": "^2.0.2",
-    "node-emoji": "^1.10.0",
     "nprogress": "^0.2.0",
     "optimize-css-assets-webpack-plugin": "^5.0.1",
     "postcss": "^7.0.6",

--- a/scripts/postinstall.js
+++ b/scripts/postinstall.js
@@ -1,6 +1,3 @@
-/* eslint-disable max-len */
-const emoji = require('node-emoji');
-
 const env = process.env;
 const ADBLOCK = is(env.ADBLOCK);
 const CI = is(env.CI);
@@ -23,16 +20,11 @@ if (!ADBLOCK && !CI && !DISABLE_OPENCOLLECTIVE && !SILENT) {
   );
   log(
     '\u001B[96mThe project needs your help! Please consider supporting of ant-design-vue on Open Collective or Patreon: ' +
-      emoji.get('pray') +
       '\u001B[0m',
   );
   log('\u001B[96m>\u001B[94m https://opencollective.com/ant-design-vue \u001B[0m');
   log('\u001B[96m>\u001B[94m https://www.patreon.com/tangjinzhou \u001B[0m\n');
-  log(
-    '\u001B[96m更多赞助方式(支付宝、微信、Paypal)请查看如下链接: ' +
-      emoji.get('pray') +
-      '\u001B[0m',
-  );
+  log('\u001B[96m更多赞助方式(支付宝、微信、Paypal)请查看如下链接: ' + '\u001B[0m');
   log('\u001B[96m>\u001B[94m https://www.antdv.com/docs/vue/sponsor-cn \u001B[0m\n');
   log('\u001B[96mAnt Design Vue 官方网站: \u001B[0m');
   log('\u001B[96m>\u001B[94m https://www.antdv.com/ \u001B[0m\n');

